index.ts

```diff
@@ -20,14 +21,18 @@ const program = new Command();
  * @param  {Array}  exceptionIds    List of vulnerability IDs to exclude
  * @param  {Array} modulesToIgnore  List of vulnerable modules to ignore in audit results
  * @param  {Array} columnsToInclude List of columns to include in audit results
+ * @param  {Boolean} outputJson     Output audit report in JSON format
+ * @param  {Array} exceptionsReport Exceptions report
  */
-export function callback(
+export const callback: HandleInputCallback = (
   auditCommand: string,
   auditLevel: AuditLevel,
   exceptionIds: string[],
   modulesToIgnore: string[],
   columnsToInclude: string[],
-): void {
+  outputJson: boolean,
+  exceptionsReport: string[][],
+): void => {
```

Comment:
is there a reason functions dont use a single object argument? This could be neater but I noticed all other functions provide individual args like this?